### DISCUSSION TITLE
Is there some way to make forge use tiled vae by default?

### DISCUSSION THREAD

**Discussion by Postmoderncaliban** [lllyasviel/stable-diffusion-webui-forge#365] [Q&A] [answered]
Sdxl always runs out and has to retry with tiled, yet it always makes this unnecessary step of trying normal vae first. I want it to default to tiled vae

**Comment by freecoderwaifu** [comment]
Set it as default in Settings > VAE.

Set any other UI settings you want as defaults, this includes any of the accordions like FreeU, Kohya HRFix, Hiresfix, etc. and also any of the other tabs like img2img.

Then go to Settings > Defaults > View changes, confirm that those values are the ones that you want to be applied by default when you launch the UI, then hit Apply.

**Reply by Postmoderncaliban** [reply]
![Screenshot from 2024-02-22 20-01-47](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/159130009/59b482d7-a83e-402d-ab20-fb77ac7f151e)
Do i need to enable advanced settings or something? I don't have an option for changing to tiled vae show up.

**Reply by freecoderwaifu** [reply]
If you have `sd_vae ` in Settings > Quicksettings list, which I think is the default in Forge, it won't show there, but using Settings > Defaults should make sure whichever VAE you have currently set will be set as default, so that's all you should need to do. 

If you have `sd_vae ` in Settings > Settings in UI > Settings for txt2img/img2img, it will show there in Settings > VAE, and will also show below Seed in the main UI tab, instead of the top of the UI. In that case, you have to set the VAE you want as default both in Settings and also in Defaults. You probably don't need to worry about this second option.

**Reply by Postmoderncaliban** [reply]
I don't think you understand my initial question. I don't want sd to default to some specific vae, but want it to use the tiled vae approach that auto-triggers when running out of vram in the decode stage for all generations, without trying the normal vae decoding first.

**Reply by freecoderwaifu** [reply, marked as answer]
Seems the activation is automatic and there's currently no way to force it to always on. There's a Feature Request about it.

https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/191#issuecomment-1958788984

**Comment by strawberrymelonpanda** [comment]
As I said in #191, I'd very much like to see this. 

I have system memory fallback turned off in Nvidia settings, and sometimes Tiled VAE isn't activated when it should be with HighRes. For example, I can Highres pass an SDXL image to 2x, but cannot to 1.5x! Why? Tiled VAE kicks in for 2x, but for 1.5x it does not and I get an OOM error.
